README.md

```diff
@@ -1 +1,32 @@
-# hw01
\ No newline at end of file
+# hw01
+
+My name is Dani Valverde. I'm a second-year in the College majoring in Statistics or Sociology, not sure which one yet. When I'm not in class, I'm involved in business RSOs on campus and I have an off-campus job coaching youth wrestling. I've lived in four different states and am moving to a fifth soon: 
+
+* Minnesota
+* Wisconsin
+* New York
+* Massachusetts
+* Arizona (moving soon!)
+
+Currently, I'm spending quarantine in Minnesota with my mom and my brother. 
+
+In quarantine, my most exciting kitchen adventure has been finding the **perfect edible cookie dough recipe**. I know to some people it might sound less than appetizing, but I *love* edible cookie dough. Here's the link to it: [https://livforcake.com/edible-cookie-dough-recipe/]
```

Comment:
Better way to embed the link using Markdown syntax

```
In quarantine, my most exciting kitchen adventure has been finding the **perfect edible cookie dough recipe**. I know to some people it might sound less than appetizing, but I *love* edible cookie dough. [Here's the link to it.](https://livforcake.com/edible-cookie-dough-recipe/)
```

In quarantine, my most exciting kitchen adventure has been finding the **perfect edible cookie dough recipe**. I know to some people it might sound less than appetizing, but I *love* edible cookie dough. [Here's the link to it.](https://livforcake.com/edible-cookie-dough-recipe/)